### PR TITLE
Merging v0.1.47 into master after release

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Merge fixes done for v0.1.47 into master, so they are part of next release.

### CODE REVIEW

File: tests/test_parse_affected.py [hunk 1]
```diff
@@ -52,7 +52,11 @@ def main():
 def parse_affected(cur_dir, env_yaml):
     for rule_dir in ssg.rules.find_rule_dirs(cur_dir):
         rule_path = os.path.join(rule_dir, "rule.yml")
-        rule = ssg.build_yaml.Rule.from_yaml(rule_path, env_yaml)
+        try:
+            rule = ssg.build_yaml.Rule.from_yaml(rule_path, env_yaml)
+        except ssg.build_yaml.DocumentationNotComplete:
+            # Happens on non-debug build when a rule is "documentation-incomplete"
+            continue
```
Comment (yuumasato) [inline tests/test_parse_affected.py]: @evgenyz @matejak There were conflicts from #4945 and #4962. 
Please double check that I resolved it correctly.

**Comment by redhatrises** [comment]
Why is this a separate branch from the existing stabilization branch?

**Comment by jan-cerny** [comment]
@yuumasato It seems that commits have been squashed which means we lost commit history.

**Comment by yuumasato** [comment]
> Why is this a separate branch from the existing stabilization branch?

My mistake, sorry.

**Comment by yuumasato** [comment]
> @yuumasato It seems that commits have been squashed which means we lost commit history.

@jan-cerny As far as I understand no history was lost. This was a merge commit of two diverging branches.

File: tests/test_parse_affected.py (hunk 1 above)
Comment (ggbecker) [reply tests/test_parse_affected.py]: it looks like it is correct, I added this piece of code actually.

**Pull request closed without merging**